bash/scripts/commit_description_parser.sh

```diff
@@ -49,7 +49,7 @@ get-most-recent-pr-commit() {
 
   curl \
   -sSL \
-  --user deis-admin:"\${GITHUB_ACCESS_TOKEN}" \
+  --user deis-admin:"${GITHUB_ACCESS_TOKEN}" \
```

Comment:
@helgi I believe this was the "derp" change needed (used to escaping `$` in groovy scripts in this repo but we do not want to escape it in the bash environment).  Still, I think the other refactor change in this PR is worthwhile (placing credentials ID in one location whenever it needs to be updated in the future).  If agreed, leet's move this through.  Otherwise, I can close this and only PR the change in this file.